src/index.Rmd

```diff
@@ -23,23 +23,33 @@ survival analysis to illustrate the main features of the package.
 Enjoy reading!
 
 ## Quick start
-A simple stratified cross-validation of [linear discriminant analysis](&MASS::lda) with [%mlr].
+Here we show the [%mlr] workflow to train, make predictions, and evaluate a learner on a classification problem. It consists of 5 basic steps that are independent of the particular learning problem and method. 
```

Comment:
Instead of "it consists...": "We walk through 5 basic steps that work on any learning problem or method supported by `mlr`."